src/System.Management.Automation/System.Management.Automation.csproj

```diff
@@ -26,6 +26,7 @@
     <PackageReference Include="System.Security.Cryptography.Pkcs" Version="5.0.0-preview.5.20278.1" />
     <PackageReference Include="System.Security.Permissions" Version="5.0.0-preview.5.20278.1" />
     <PackageReference Include="System.Text.Encoding.CodePages" Version="5.0.0-preview.5.20278.1" />
+    <PackageReference Include="System.Windows.Extensions" Version="5.0.0-preview.5.20278.1" />
```

Comment:
It looks to me `System.Windows.Extensions` has already been included:
```powershell
PS:84> dir $PSHOME\System.Windows.Extensions.dll

    Directory: C:\Program Files\PowerShell\7

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a---           4/15/2020  1:01 AM         118152 System.Windows.Extensions.dll
```

Reply:
Maybe better to have explicit reference if .Net team changes something in dependencies?

Reply:
That would be a breaking change from their side.
Explicitly referencing the package might result in conflicting package versions. I prefer to depending on the one comes from the .NET 5 framework.

Reply:
Removed.